reviews/serializers.py

```diff
@@ -20,23 +19,43 @@
 
 
 # Pseudo-class to hide the creator field if it shouldn't be shown
-def IfCanViewLogCreator(field_cls):
-    class IfCanViewLogCreatorField(field_cls):
+def UnlessCommentsAnonymous(field_cls):
```

Comment:
Is "Unless" fields used other places? It seems a little strange.

Reply:
That does seem a little better; less logic to think through. Up to you though, I don't feel strongly either way.